examples/advanced/variables.tf

```diff
@@ -25,7 +26,7 @@ variable "access_tags" {
 variable "region" {
   description = "Region where resources will be created"
   type        = string
-
+  default     = "us-south"
```

Comment:
default value was purposely removed from examples - please remove it. See https://github.ibm.com/GoldenEye/issues/issues/5505